src/Storage/Commands.Storage/Common/Cmdlet/DisableAzureStorageDeleteRetentionPolicy.cs

```diff
@@ -0,0 +1,82 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet
+{
+    using Microsoft.WindowsAzure.Commands.Storage.Model.ResourceModel;
+    using Microsoft.WindowsAzure.Storage.Shared.Protocol;
+    using System;
+    using System.Management.Automation;
+    using System.Security.Permissions;
+
+    /// <summary>
+    /// Disable azure storage service DeleteRetentionPolicy, currently only available on Blob service
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Disable, StorageNouns.ServiceDeleteRetentionPolicy, SupportsShouldProcess = true),
+        OutputType(typeof(PSDeleteRetentionPolicy))]
+    public class DisableAzureStorageServiceDeleteRetentionPolicyCommand : StorageCloudBlobCmdletBase
```

Comment:
I think the comm and name should be shorter: Disable-AzureStorageSoftDelete

Reply:
@markcowl 
The name of the cmdlets Enable/Disable-AzureStorageServiceDeleteRetentionPolicy are reviewed and raised from your team, and feature team also agreed for that.
The current name is aligned with the rest and XSCL API, so customer who have looked into rest doc might can find it more easily.

Do you really want to change it?

Reply:
As discussed with Mark, the alias for the cmdlets is added: Disable-AzureStorageSoftDelete